screens/CameraRecording.tsx

```diff
@@ -174,7 +176,7 @@ const CameraRecording: FunctionComponent<CameraRecordingProps> = ({
         </>
       )}
       {!shouldShowCamera && (
-        <SavedOverlay finished={finished} overlay={overlay} />
+        <SavedOverlay finished={finished} overlay={overlay} name={name} />
```

Comment:
We pass in name into SavedOverlay but SavedOverlay never uses it?

Reply:
Hmm so it takes in the CameraRecordingProps. Perhaps we can make "name" optional within the CameraRecordingProps?

Reply:
This was a feat of laziness. I at one point just didn't want to make a new interface, so I never updated the SavedOverlay with a new one, and now this has grown, and SavedOverlay is just along for the ride.

Reply:
I will change this.